### PR TITLE
Content Item Skeletons

### PR DESCRIPTION
**Pull request by MichaelKovalchuk**
skeleton-demo

### CODE REVIEW

**Comment by MichaelKovalchuk** [comment]
I added artificial 5 second delay to make the GIF

**Comment by AdmiralCoco** [comment]
What do you think about:
- Increasing the size (height?) of the name/text skeletons? Looks a bit small
- Pulsating animation: is this present elsewhere in the LMS? (I believe CS/Publish/Disco do not have animations - might want to keep it consistent)

File: src/components/content-list.js [hunk 1]
```diff
@@ -117,10 +125,44 @@ class ContentList extends DependencyRequester(InternalLocalizeMixin(LitElement))
...
 			</d2l-list>
 		`;
 	}
 
+	renderContentItemSkeletons(count) {
+		const skeletonTemplates = [];
+		for (let i = 0; i < count; i++) {
+			skeletonTemplates.push(html`
+			<d2l-list-item class="d2l-body-compact" ?selectable=${this.contentItemsSelectable} disabled>
+				<content-skeleton width="42px" height="42px" slot="illustration"></content-skeleton>
```
Comment (AdmiralCoco) [inline src/components/content-list.js]: Might be wrong here, but are we able to to:
- in `content-skeleton`, set `max-height/width` to 100%
- style the `content-skeletons` here, through CSS (e.g., ` .skeleton-text { width: 95% }`)

I don't know if we want to pass styles as properties along, unless we have to

**Comment by MichaelKovalchuk** [comment]
@AdmiralCoco Pulsating animation and skeleton height were taken from here: http://design.d2l/patterns/skeleton-loading/

I like the idea of moving styles out of the attributes

**Pull request merged**